packages/tabs/test/lion-tabs.test.js

```diff
@@ -15,6 +15,8 @@ const basicTabs = html`
     <div slot="panel">panel 2</div>
     <button slot="tab">tab 3</button>
     <div slot="panel">panel 3</div>
+    <button slot="tab">tab 4</button>
+    <div slot="panel">panel 4</div>
```

Comment:
Added another tab, in case some of the test expects being adjusted to +1, this explains those diffs ;)

Reply:
Thanks 😄